config.template.toml

```diff
@@ -159,8 +172,12 @@ model = "gpt-4o"
 # Memory maximum threads
 #memory_max_threads = 2
 
-# Name of the agent
-#name = "CodeActAgent"
+# LLM config group to use
+#llm_config = 'llm'
+
+[BrowsingAgent]
+# e.g. use a cheaper model for BrowsingAgent
+llm_config = 'gpt3_llm'
```

Comment:
I don't think `BrowsingAgent` would work as well with gpt-3. I know this is supposed to be an example, but some people might take this as a recommendation, so perhaps we could keep it with gpt-4 by default.